tpkg/bad_cpu_affinity.tdir/bad_cpu_affinity.pre

```diff
@@ -0,0 +1,18 @@
+# #-- bad_cpu_affinity.pre--#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+. ../common.sh
+
+# start NSD
+get_random_port 1
+NSD_PORT=$RND_PORT
+echo port: $NSD_PORT
+
+NUM_CPUS="$(cpu_count)"
```

Comment:
This value does not exist for me.  On freeBSD it seems to be from `sysctl -n hw.ncpu` that prints something like `4` or `sysctl: cannot stat /proc/sys/hw/ncpu: No such file or directory` on Linux.  On Linux, /proc/cpuinfo contains lines of text, and I think you need `grep "^processor" /proc/cpuinfo  | wc -l` that also prints the plain number.

Reply:
Ok. Good to know it doesn't exist everywhere. I just grabbed something that existed on my Fedora machine. Indeed, the `grep` version might work on more systems, I'll use that.

Reply:
Sorry about that, nproc works fine for me too.  And I had not read the common.sh part yet where you have the shell subroutine for that.  So you can just use that.